### ISSUE TITLE
sh: gwc: command not found

### ISSUE THREAD

**Issue by Sugarbank** [junyanz/CycleGAN#20]
Nicks-iMac:CycleGAN-master nickrabone$ DATA_ROOT=./datasets/ae_photos name=style_cezanne_pretrained model=one_direction_test phase=test loadSize=256 fineSize=256 resize_or_crop="scale_width" gpu=0 th test.lua
------------------- Options -------------------	
                DATA_ROOT: ./datasets/ae_photos	
               align_data: 0	
             aspect_ratio: 1	
                batchSize: 1	
          checkpoints_dir: ./checkpoints	
           continue_train: 1	
                    cudnn: 1	
                  display: 1	
               display_id: 200	
                 fineSize: 256	
                     flip: 0	
                      gpu: 0	
                 how_many: all	
                 input_nc: 3	
                 loadSize: 256	
                    model: one_direction_test	
                 nThreads: 1	
                     name: style_cezanne_pretrained	
                     norm: instance	
                output_nc: 3	
                    phase: test	
               preprocess: regular	
           resize_or_crop: scale_width	
              results_dir: ./results/	
           serial_batches: 1	
                     test: 1	
          which_direction: AtoB	
              which_epoch: latest	
-----------------------------------------------	
CPU Mode	
DataLoader UnalignedDataLoader was created.	
Starting donkey with id: 1 seed: 8835
table: 0x105ea028
table: 0x105d54f0
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
/tmp/lua_IlsKnP: line 1: gfind: command not found
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
cmd..gwc -L '/tmp/lua_ZH3IMp' |gcut -f1 -d' '
sh: gwc: command not found
/Users/nickrabone/torch/install/bin/luajit: ...ckrabone/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 1 callback] ...esktop/StylaRender/APPS/CycleGAN-master/data/dataset.lua:211: attempt to perform arithmetic on a nil value

**Comment by Sugarbank** [comment]
fixed that error by installing 
brew install coreutils

however now getting this error although there are images in the directory
"Could not find any image file in the given input paths"

Check below for output

Nicks-iMac:CycleGAN-master nickrabone$ DATA_ROOT=./datasets/ae_photos name=style_cezanne_pretrained model=one_direction_test phase=test loadSize=256 fineSize=256 resize_or_crop="scale_width" gpu=0 th test.lua
------------------- Options -------------------	
                DATA_ROOT: ./datasets/ae_photos	
               align_data: 0	
             aspect_ratio: 1	
                batchSize: 1	
          checkpoints_dir: ./checkpoints	
           continue_train: 1	
                    cudnn: 1	
                  display: 1	
               display_id: 200	
                 fineSize: 256	
                     flip: 0	
                      gpu: 0	
                 how_many: all	
                 input_nc: 3	
                 loadSize: 256	
                    model: one_direction_test	
                 nThreads: 1	
                     name: style_cezanne_pretrained	
                     norm: instance	
                output_nc: 3	
                    phase: test	
               preprocess: regular	
           resize_or_crop: scale_width	
              results_dir: ./results/	
           serial_batches: 1	
                     test: 1	
          which_direction: AtoB	
              which_epoch: latest	
-----------------------------------------------	
CPU Mode	
DataLoader UnalignedDataLoader was created.	
Starting donkey with id: 1 seed: 5341
table: 0x0150c068
table: 0x014ce638
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
/tmp/lua_4hxz1y: line 1: gfind: command not found
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
cmd..gwc -L '/tmp/lua_5nLd8m' |gcut -f1 -d' '
/Users/nickrabone/torch/install/bin/luajit: ...ckrabone/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 1 callback] ...esktop/StylaRender/APPS/CycleGAN-master/data/dataset.lua:215: Could not find any image file in the given input paths
stack traceback:
	[C]: in function 'assert'
	...esktop/StylaRender/APPS/CycleGAN-master/data/dataset.lua:215: in function '__init'
	...rs/nickrabone/torch/install/share/lua/5.1/torch/init.lua:91: in function <...rs/nickrabone/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'dataLoader'
	.../StylaRender/APPS/CycleGAN-master/data/donkey_folder.lua:205: in main chunk
	[C]: in function 'dofile'
	...e/Desktop/StylaRender/APPS/CycleGAN-master/data/data.lua:39: in function <...e/Desktop/StylaRender/APPS/CycleGAN-master/data/data.lua:29>
	[C]: in function 'xpcall'
	...ckrabone/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
	...nickrabone/torch/install/share/lua/5.1/threads/queue.lua:65: in function <...nickrabone/torch/install/share/lua/5.1/threads/queue.lua:41>
	[C]: in function 'pcall'
	...nickrabone/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
stack traceback:
	[C]: in function 'error'
	...ckrabone/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
	...ckrabone/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
	...ckrabone/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
	...ckrabone/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
	...e/Desktop/StylaRender/APPS/CycleGAN-master/data/data.lua:27: in function 'new'
	...ktop/StylaRender/APPS/CycleGAN-master/data/data_util.lua:17: in function 'load_dataset'
	./data/unaligned_data_loader.lua:24: in function 'Initialize'
	test.lua:47: in main chunk
	[C]: in function 'dofile'
	...bone/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x0100974a00

**Comment by subzerofun** [comment]
Try putting `sudo` before the command, it helped me with an error i got while trying out the other training examples (i'm on OSX 10.12).

**Comment by whatru** [comment]
@Sugarbank to fix gfind: command not found
please by installing:
brew install findutils

now I find en error:
unknown Torch class <torch.CudaTensor>
I have no idea either
